app/controllers/articles_controller.rb

```diff
@@ -46,14 +48,20 @@ def update
 	end
 
 	def destroy
-		@article = Article.find(params[:id])
-		@article.destroy
+   @article = Article.find(params[:id])
+		if current_user.id != @article.user_id
 
-		if @article.destroy
-			redirect_to root_path, notice: "Article was successfully deleted"
+			redirect_to @article, :notice => "Invalid user"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.